run.php

```diff
@@ -6,51 +6,54 @@
  * @copyright Copyright (c) 2017 Antonios Pavlakis
  * @license   https://github.com/pavlakis/notts-digital/blob/master/LICENSE (BSD 3-Clause License)
  */
+use NottsDigital\Event\EventInterface;
+
 date_default_timezone_set('Europe/London');
 
 require __DIR__ . '/vendor/autoload.php';
 require __DIR__ . '/app/dependencies.php';
 
 
-$groups = $container['groups'];
+$groupsContainer = $container['groups.container'];
 
 $request = Zend\Diactoros\ServerRequestFactory::fromGlobals();
 
 // index.php?group=PHPMinds
 $params = $request->getQueryParams();
 $group = '';
 
-// return empty json by default
-$response = new Zend\Diactoros\Response\JsonResponse([
-    'group'     => '',
-    'subject'   => '',
-    'description'   => '',
-    'date_time' => '',
-    'location'  => '',
-    'event_url' => '',
-    'iso_date' => '',
-    'next_event' => []
-]);
+try {
 
-if (array_key_exists('group', $params)) {
+    if (!array_key_exists('group', $params)) {
+        throw new \Exception('Group name required.', 400);
+    }
 
     $group = $params['group'];
-    $eventType = '';
-
-    foreach ($groups as $type => $events) {
-        if (array_key_exists($group, $events)) {
-            $eventType = $type;
-            break;
-        }
+    if (!$groupsContainer->has($group)) {
+        throw new \Exception('Group does not exist', 404);
     }
 
-    if ($container->offsetExists('event.' . $eventType) === true) {
-        $event = $container['event.' . $eventType];
-        $event->getByGroup($group);
+    $eventPlatform = $groupsContainer->get($group);
 
-        $response = new Zend\Diactoros\Response\JsonResponse($event->toArray(), 200, [], JSON_PRETTY_PRINT);
+    if (!$container->offsetExists('event.' . $eventPlatform)) {
```

Comment:
phrase Event platform does not exist - could this be more user friendly as it gets output to json?